modules/ingest-geoip/src/yamlRestTest/resources/rest-api-spec/test/ingest_geoip/40_geoip_databases.yml

```diff
@@ -1,8 +1,24 @@
+---
 setup:
   - requires:
       cluster_features: ["geoip.downloader.database.configuration", "get_database_configuration_action.multi_node"]
       reason: "geoip downloader database configuration APIs added in 8.15, and updated in 8.16 to return more results"
 
+---
+teardown:
+  - do:
+      ingest.delete_ip_location_database:
+        id: "my_database_1"
+        ignore: 404
+  - do:
+      ingest.delete_ip_location_database:
+        id: "my_database_2"
+        ignore: 404
+  - do:
+      ingest.delete_ip_location_database:
+        id: "my_database_3"
```

Comment:
This one is never created in this file.

Reply:
Good eye!

Reply:
Fixed in 116394a1fd572a852d517c60b658154a0b37b33a.